CONTRIBUTING.md

```diff
@@ -205,6 +206,45 @@ To start Camunda:
 
 If you need to change any of the default configuration values for Camunda, the configuration files are available in `CAMUNDA_HOME/config`. After updating the configuration you need to restart any running services for the changes to take effect.
 
+#### Run via IntelliJ
+
+You can run the Camunda distribution via IntelliJ for development purposes.
+
+1. Run a **full build** of the project via Maven as [described above](#build). Make sure to use
+   `install` as we need to depend on 2 locally built artifacts.
+2. Open the project in IntelliJ.
+3. Currently, there is an issue with the `com.auth0.mvc-auth-commons` library being transformed to
+   a Jakarta-compatible version during the Maven build process, but this change not being picked up
+   by IntelliJ.
+   - To address this, open the _Project Structure_ dialog and add the following 2 libraries to the
+     classpath of your JDK (`<version>` being the current snapshot version, e.g. `8.8.0`):
+       - `$HOME/.m2/repository/io/camunda/operate-mvc-auth-commons/<version>-SNAPSHOT/operate-mvc-auth-commons-<version>-SNAPSHOT.jar`
+       - `$HOME/.m2/repository/io/camunda/tasklist-mvc-auth-commons/<version>-SNAPSHOT/tasklist-mvc-auth-commons-<version>-SNAPSHOT.jar`
+   - Example: ![intellij-module-settings.png](docs/assets/intellij-module-settings.png)
+   - If you receive an error on saving the project structure settings regarding the `zeebe-gateway-protocol`
```

Comment:
🔧 that's actually not related to auth-commons, I also didn't encounter it yet 🤔 Wdyt about listing such potential issues in a troubleshooting sub-section?

Reply:
Good point, moved it to a troubleshooting section underneath the actual guide 👍 

> I also didn't encounter it yet

Interesting, I started to get this every time I open the dialog some time ago (also tried with a new checkout/new project) as soon as I hit apply on the project structure dialog. 🤔 Would it make sense to open an issue to investigate this at some point?

![image](https://github.com/user-attachments/assets/458e4532-adc1-44d6-b50e-3c1ec5771692)